src/Controls/src/Core/Compatibility/Handlers/ListView/iOS/CellRenderer.cs

```diff
@@ -135,44 +133,9 @@ protected void UpdateBackground(UITableViewCell tableViewCell, Cell cell)
 			SetBackgroundColor(tableViewCell, cell, uiBgColor);
 		}
 
+		[Obsolete("The ForceUpdateSizeRequested event is now managed by the command mapper, so it's not necessary to invoke this event manually.")]
```

Comment:
This method was just a really complicated way of wiring up to `_onForceUpdateSizeRequested` and making sure to unwire from it. 

We can just achieve this same thing through the `commandmapper` now